packages/amplify-provider-awscloudformation/src/aws-utils/aws-amplify.js

```diff
@@ -1,21 +1,7 @@
 const aws = require('aws-sdk');
 const proxyAgent = require('proxy-agent');
 const configurationManager = require('../configuration-manager');
-
-const amplifyServiceRegions = [
-  'us-east-1',
-  'us-east-2',
-  'us-west-2',
-  'eu-west-1',
-  'eu-west-2',
-  'eu-central-1',
-  'ap-northeast-1',
-  'ap-northeast-2',
-  'ap-south-1',
-  'ap-southeast-1',
-  'ap-southeast-2',
-  'ca-central-1',
-];
+const { regions: amplifyServiceRegions } = require('../aws-regions');
```

Comment:
Is this list identical to what we had?

Reply:
Yes, although not sorted in the same order. [Here](https://github.com/aws-amplify/amplify-cli/blob/d55917fd83140817a4447b3def1736f75142df44/packages/amplify-provider-awscloudformation/src/aws-regions.js#L4-L17) is where they come from.